module-2/backend/week_four.md

```diff
@@ -10,14 +10,57 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+
+  A cookie is a token that holds user information that can be recognized by the requesting server. 
+
 * What’s the difference between a session and a cookie?
+
+  A session is a more secure version of a cookie (encrypted); both are stored on the client side, and act like reminders to the server with details about the client. They are ways of maintaining state between requests. 
+  
 * What’s a flash and when do you want to use flashes?
+  
+  A flash is a message that pops up on a user's screen, often to let them know about the success or failure of a controller action (you successfully edited/created/deleted some record). We iterate over the flash hash, which has keys like "Notice" and "Alert", which we've set to some value, and we post that message through our view. A flash is a self-deleting portion of a the session. Although I'm not really sure what that means as I write it. 
+
+
 * Why do people say “HTTP is stateless”?
+
+  Because it's not holding onto any information in between server calls (server calls? requests? what's the difference?)
```

Comment:
I would consider `server call` a synonym (maybe a little less formal) for `request`.